syntax = "proto3";

package tc_serialization;

message RoutingSettings{
    int32 bus_wait_time = 1;
    double bus_velocity = 2;
}